src/cocotb/handle.py

```diff
@@ -296,6 +303,30 @@ def _discover_all(self) -> None:
 
         self._discovered = True
 
+    def _get(
+        self, key: str, discovery_method: GPIDiscovery = GPIDiscovery.AUTO
+    ) -> Optional[simulator.gpi_sim_hdl]:
+        """Query the simulator for an object with the specified *key*.
+
+        Like Pythons native dict get-function, this returns None if the object
+        is not found instead of raising an AttributeError.
+
+        If extended identifiers are needed simply add a ``\\`` character before and after the name.
```

Comment:
```suggestion
```
This is only correct for VHDL. Best to just not include it.

Reply:
should this be mentioned nowhere? I just included it because _get will kind of replace _id()

remove it completely or mention it's just for VHDL?